src/briefcase/commands/create.py

```diff
@@ -575,6 +579,86 @@ def install_image(self, role, variant, size, source, target):
                     )
                 )
 
+    """
+    General Notes
+
+    I have omitted anything platform-specific, including the working Apple code, as I think it would be best to get feedback on this inital structure. I am unsure how to integrate platform-specific features. Would I start with going to each platform's dev page and collecting the filesize & name specified?
+    
+    Nothing with splash screens
+    
+    Nothing tested
+    
+    Feedback Requested:
+    
+    Should multiple image types as input be supported? jpgs or other?
+    
+    Do the function names, method signatures and return types seem on the right track?
+    
+    What variable or variables would give me information about the target platform?
+    
+    I'm open to feedback on anything
+    
```

Comment:
General note - this sort of comment should be in the PR comment, not the code itself.